### DISCUSSION TITLE
genes with open promoter

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1309] [Questions / Documentation] [answered]
Hello again!

I know that we can extract peak type from ArchR, let's say that I want to look at genes with open promoters. I was wondering if there is a way to assess whether the peak in the promoter region is significantly accesible? Just by exploring individual genes I do see that some have high peaks at the promoter, and some don't, and I basically want to see if there are any statistics assigned to these differences.

Thank you!

**Comment by rcorces** [comment, marked as answer]
Nothing provided by ArchR. You would have to decide what level of signal corresponds to "accessible" and what level corresponds to "not accessible"

**Reply by rojinsafavi** [reply]
Thanks, Ryan, could you please point me to where I can find the normalized accessibility level of each region? Do you suggest using the peak calling matrix directly?

**Reply by rcorces** [reply]
I dont have a perfect way of doing this yet but it is on my radar as a missing piece of ArchR. Assuming we are talking about grouped data (pseudobulks), I would use `getGroupSE()` to get a counts matrix. The missing piece is that the counts in this matrix would need to be manually normalized and I dont think the current normalizations provided by the function are entirely appropriate. I would normalize each group by the sum of `ReadsInTSS` for all cells in that group. Then log2 transform plus a prior count for ex:

```
preprocessCore::normalize.quantiles(edgeR::cpm(counts_mat,log=TRUE,prior.count=5))
```

**Reply by rojinsafavi** [reply]
Thank you so much again!
